config/Shared/config_propel.php

```diff
@@ -84,7 +84,6 @@
         'sqlDir' => APPLICATION_ROOT_DIR . '/src/Orm/Propel/Sql',
         'migrationDir' => APPLICATION_ROOT_DIR . '/src/Orm/Propel/Migration_' . $config[PropelConstants::ZED_DB_ENGINE],
         'schemaDir' => APPLICATION_ROOT_DIR . '/src/Orm/Propel/Schema',
-        'loaderScriptDir' => APPLICATION_ROOT_DIR . '/src/Orm/Propel/generated-conf/',
```

Comment:
b2c-mp has the same issue, do we have a PR for it? @fsmeier

Reply:
@asmarovydlo oh you are right. I thought i checked all the other demo-shops but i somehow overlooked the line in b2c-mp. PR created https://github.com/spryker-shop/b2c-demo-marketplace/pull/201

Reply:
Thanks @fsmeier , I'll merge it Travis will be Ok.